torch/_refs/nn/functional/__init__.py

```diff
@@ -12,9 +12,42 @@
 from typing import Optional
 
 __all__ = [
+    "celu",
     "elu",
 ]
 
+# celu is implemented specially because it has an alpha argument
+# celu is very similar to elu
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("a",),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.OP_MATH,
+)
+def celu(
+    a: TensorLikeType, alpha: Optional[NumberType] = None, inplace: bool = False
+) -> TensorLikeType:
+    """
+    Reference implementation of torch.nn.functional.elu
+    """
+
+    if inplace:
+        raise NotImplementedError
+
+    rhs: TensorLikeType
+    if alpha is not None:
+        python_type = utils.dtype_to_type(a.dtype)
+        if not utils.is_weakly_lesser_type(type(alpha), python_type):
+            msg = (
+                "alpha argument of type {0} cannot be safely cast to type {1}!".format(
+                    type(alpha), python_type
+                )
+            )
+            raise ValueError(msg)
+        rhs = refs.mul(alpha, refs.expm1(refs.mul(a, (1.0 / alpha))))
```

Comment:
Nice workaround for div not being implemented yet -- we should add div

Reply:
Good news -- you can use refs.true_divide here, see

https://github.com/pytorch/pytorch/blob/2f602abf1454ccb21a012a8bbfe48f448c983467/torch/_refs/__init__.py#L702